packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/ClientProvider.cs

```diff
@@ -117,14 +117,14 @@ public ClientProvider(InputClient inputClient)
             }
         }
 
-        private List<ParameterProvider>? _uriParameters;
-        internal IReadOnlyList<ParameterProvider> GetUriParameters()
+        private List<ParameterProvider>? _clientParameters;
```

Comment:
We will not propagate client parameters to operation, when generate `CreateXXXRequest` method, RestClientProvider needs to get all the client parameter (not just uri parameter) from ClientProvider. So we provide a function to get all client parameters.